webpack.config.mjs

```diff
@@ -64,18 +64,11 @@ function webpackConfig(_env, { mode = "development" }) {
     module: {
```

Comment:
I thought webpack was not needed anymore.

Reply:
what should webpack not be needed anymore? Webpack is the bundler. Babel which is being replaced here is the transpiler. Once [turbopack](https://turbo.build/pack) supports anything other than next.js or [swcpack](https://swc.rs/docs/usage/bundling) becomes usable, we can switch to that. Up until then we are stuck with webpack or vite. Also snowpack is dead.